activerecord/lib/active_record/database_configurations/hash_config.rb

```diff
@@ -158,6 +158,10 @@ def schema_dump(format = ActiveRecord.schema_format)
         end
       end
 
+      def schema_format
```

Comment:
This should be no-doc, since we don't want apps calling `db_config.schema_format`.

```suggestion
      def schema_format # :nodoc:
```